### ISSUE TITLE
Conv2DBackpropInput layer in ACL?

### ISSUE THREAD

**Issue by abhajaswal** [ARM-software/ComputeLibrary#795]
I am trying to check the support for Conv2DBackpropInput in arm compute library.
I Checked the path ComputeLibrary-19.11\arm_compute\runtime\CL\functions for required interface , but could not locate it.
Can you please let me know if this is supported in release v 19.11?

**Comment by abhajaswal** [comment]
Hi @morgolock  can you please let me know about it!

**Comment by abhajaswal** [comment]
I want to implement ACL solution for UNET model for image segmentation - Model for same is at location https://github.com/PINTO0309/TensorflowLite-UNet/tree/master/model/semanticsegmentation_frozen_person_08.pb 

This contains the Conv2DBackpropInput & Strided slice and pack operations.

Can you help me in regarding these opeartors interface files

**Comment by GeorgeARM** [comment]
Hello @abhajaswal,

ComputeLibrary is designed for inference that you won't find any training operations in place. In other words, we do not have a function to compute the gradients of a convolution.
Hope this helps.
